### DISCUSSION TITLE
Docker-compose environment variable for php.ini settings

### DISCUSSION THREAD

**Discussion by wchouser3** [docker-library/wordpress#823] [General]
how do I change:
upload_max_filesize;
post_max_size;
max_execution_size?

I couldn't find a php.ini file in the usual places when exec'ed into the container. I don't imagine that's the preferred way to do it anyway. Is there an environment variable?

Thank You!

**Comment by tianon** [comment]
See "Configuring PHP directives" from https://hub.docker.com/_/wordpress (https://github.com/docker-library/docs/tree/master/wordpress#configuring-php-directives).

**Reply by wchouser3** [reply]
Not sure why I didn't see that before. Thank you!
